extensions/expanded-opcodes/README.md

```diff
@@ -0,0 +1,87 @@
+**Requires: Base, VWI**  
+**CPUID Bit: 9**
+
+# Overview
+
+This extension adds an expanded instruction format to allow for a larger number of opcodes to be used. It also includes a relative jump and call with larger displacements.
+
+# Added Instructions
+
+The following opcodes are now defined.
+
+| First byte    | Second Byte  | Third Byte   | Comment                                                                   |
+|:--------------|:-------------|:-------------|:--------------------------------------------------------------------------|
+| `111 00 000`  | `SS CCCCCC`  | `RRR RRR MM` | Register-Register operation                                               |
+| `111 00 ???`  | `SS CCCCCC`  | `RRR RRR MM` | When `???` is not `000`, reserved                                         |
+| `111 01 000`  | `SS CCCCCC`  | `RRR IIIII`  | Register-Immediate operation                                              |
+| `111 01 ???`  | `SS CCCCCC`  | `RRR IIIII`  | When `???` is not `000`, reserved                                         |
+| `111 10 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative jump                                                       |
+| `111 11 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative call if stack extension is implemented, otherwise reserved |
+
+| Symbol | Meaning                                    |
+|--------|--------------------------------------------|
+| C      | condition code bit                         |
+| R      | register id                                |
+| I      | immediate                                  |
+| D      | signed displacement                        |
+| S      | size                                       |
+| M      | memory mode                                |
+| ?      | reserved for extensions or another purpose |
+
+## Mapping from the base ISA opcodes to this extension
+
+When the base opcode `CCCC` matches `0xxx`, the new opcode is `0 CCCC 0`
+
+When the base opcode `CCCC` matches `1xxx`, the new opcode is `01 CCCC`
+
+## Opcode mapping table
+
+| `CCCCCC` | NAME            | Operation                          | Flags  | Comment     |
+|----------|-----------------|------------------------------------|--------|-------------|
+| `000000` | `ADD`           | `A ← A + B`                        | `ZNCV` |             |
+| `000001` | `ADC`           | `A ← A + B + C`                    | `ZNCV` |             |
+| `000010` | `SUB`           | `A ← A - B`                        | `ZNCV` |             |
+| `000011` | `SBB`           | `A ← A - B - C`                    | `ZNCV` |             |
+| `000100` | `RSUB`          | `A ← B - A`                        | `ZNCV` | (1)         |
+| `000101` | `RSBB`          | `A ← B - A - C`                    | `ZNCV` |             |
+| `000110` | `CMP`           | `_ ← A - B`                        | `ZNCV` |             |
+| `000111` | `RCMP`          | `_ ← B - A`                        | `ZNCV` |             |
```

Comment:
This should not exist.

Reply:
github apparently decided to ignore my line selection on this one: specifically commenting on `rcmp`.

Reply:
Yeah, I'll remove it. I blame doing this at 3 or 4 am :stuck_out_tongue: